packages/backend/src/app/ee/platform/platform.controller.ts

```diff
@@ -0,0 +1,37 @@
+import { FastifyPluginCallbackTypebox, Type } from '@fastify/type-provider-typebox'
+import { UpdatePlatformRequestBody } from '@activepieces/ee-shared'
+import { ApId } from '@activepieces/shared'
+import { platformService } from './platform.service'
+
+export const platformController: FastifyPluginCallbackTypebox = (app, _opts, done) => {
+    app.patch('/:id', UpdatePlatformRequest, async (req) => {
+        return await platformService.update({
+            id: req.params.id,
+            userId: req.principal.id,
+            ...req.body,
+        })
+    })
+
+    app.get('/:id', GetPlatformRequest, async (req) => {
```

Comment:
Nit: both APIs required to be restricted by information from JWT token, but i assume this coming in the next pr.

Reply:
Update: I see you assert by owner Id, usually another fail safe way to make it query by owner id and id, this also prevent mistakes.